cypress/integration/jsPromisesPracticeDOM.spec.js

```diff
@@ -18,70 +18,53 @@ describe('Promises in DOM', () => {
     cy.visit('/');
   });
 
-  it('should resolve first promise on the left click', () => {
-    cy.clickButton({ button: 0 });
-    cy.checkMessageDisplayed(firstResolvedMsg);
-    // NOTE: waiting for reject
-    // eslint-disable-next-line cypress/no-unnecessary-waiting
-    cy.wait(3000);
-    cy.checkMessageDisplayed(firstRejectedMsg).should('not.exist');
-  });
+  describe('firstPromise', () => {
+    it('should be resolved on the left click', () => {
+      cy.clickButton({ button: 0 });
+      cy.checkMessageDisplayed(firstResolvedMsg);
```

Comment:
Are you sure you need these custom commands?
We could just create a page object with notification and check it's text